### PR TITLE
[NPC] Humilitie - Dock NPC in Selbina changes

### PR DESCRIPTION
**Pull request by kaincenteno**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

* Adds triggers for Selbina ferry arriving and departing

* Adds the interval between ships

* Adds offset from Epoch for boat arriving and leaving.

* Adds movement point and rotation for NPC at selbina dock.

* Adds text IDs for Humilitie to speak when the ferry arrives and leaves; along hooking up a periodic trigger to her.

Thank you @Daywhite83  for providing the position for NPC when moving at arriving and departing times.

## Steps to test these changes

Go to Selbina dock and look at NPC talk and walk when ferry arrives/departs

image

image

### CODE REVIEW

File: scripts/zones/Selbina/IDs.lua [hunk 1]
```diff
@@ -23,6 +23,8 @@ zones[xi.zone.SELBINA] =
         FISHING_MESSAGE_OFFSET        = 6565, -- You can't fish here.
         NOMAD_MOOGLE_DIALOG           = 6665, -- I'm a traveling moogle, kupo. I help adventurers in the Outlands access items they have stored in a Mog House elsewhere, kupo.
         SUBJOB_UNLOCKED               = 6868, -- You can now designate a support job.
+        FERRY_DEPARTING               = 7019, -- All aboard!≺Prompt≻
```
Comment (WinterSolstice8) [inline scripts/zones/Selbina/IDs.lua]: Update extractor says this should be only `All aboard!` with an ID of `7016`

File: scripts/zones/Selbina/IDs.lua [hunk 2]
```diff
@@ -23,6 +23,8 @@ zones[xi.zone.SELBINA] =
...
+        FERRY_ARRIVING                = 7017, -- Attention, passengers! The ship to Mhaura is here and ready to board!≺Prompt≻
```
Comment (WinterSolstice8) [inline scripts/zones/Selbina/IDs.lua]: Update extractor says this should only be `Attention, passengers! The ship to Mhaura is here and ready to board!` with an id of `7014`

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]

**Comment by WinterSolstice8** [comment]
Out of scope for this PR but there's a couple other messages that look like they could be added if we know their conditions, provided something else isnt already using them:

```
7015 : Attention, passengers! The ship is about to depart. Please make haste!
7017 : You'll have to wait a while for the next boat.
7018 : This ship is bound for Mhaura.
```

File: scripts/zones/Selbina/IDs.lua (hunk 1 above)
Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: Where can i find the update extractor? I got this from POLUtils?

Comment (WinterSolstice8) [reply scripts/zones/Selbina/IDs.lua]: https://github.com/LandSandBoat/UpdateExtractor

just make sure to update your client if you haven't already, there was a patch a few days ago. The comments that go in the lines there need to match update extractor's output, not polutils

Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: removed the <prompt>

Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: it shows as 7019 on Octobers update 

  </thing>
  <thing type="DialogTableEntry">
    <field name="index">7019</field>
    <field name="text">All aboard!≺Prompt≻
</field>

Comment (WinterSolstice8) [reply scripts/zones/Selbina/IDs.lua]: One sec, my polutils agrees. I think update extractor might've used some old cached files.

Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: sure let me know.

About the <prompt> part. should that be removed anyway?

Comment (WinterSolstice8) [reply scripts/zones/Selbina/IDs.lua]: Yup, it looks like it was using stale files. New run agrees with this PR. And yes the `<Prompt>` should be removed

**Review by WinterSolstice8 — APPROVED** [review]

File: scripts/zones/Selbina/IDs.lua (hunk 1 above)
Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: already removed =) both of them

**Review by claywar — APPROVED** [review]

File: scripts/zones/Selbina/IDs.lua [hunk 3]
```diff
@@ -23,6 +23,8 @@ zones[xi.zone.SELBINA] =
         FISHING_MESSAGE_OFFSET        = 6565, -- You can't fish here.
         NOMAD_MOOGLE_DIALOG           = 6665, -- I'm a traveling moogle, kupo. I help adventurers in the Outlands access items they have stored in a Mog House elsewhere, kupo.
         SUBJOB_UNLOCKED               = 6868, -- You can now designate a support job.
+        FERRY_DEPARTING               = 7019, -- All aboard!
```
Comment (claywar) [inline scripts/zones/Selbina/IDs.lua]: Could you please put these additions in sequential order?

Comment (kaincenteno) [reply scripts/zones/Selbina/IDs.lua]: done :)

**Review by claywar — APPROVED** [review]

**Pull request merged**